docusaurus-plugin-moonwave/src/index.js

```diff
@@ -1,11 +1,19 @@
 const path = require("path")
 const fs = require("fs")
 const { promisify } = require("util")
+const { create } = require("domain")
 const exec = promisify(require("child_process").exec)
 
+const SECTIONS = ["types", "properties", "functions"]
+
+const capitalize = (text) => text[0].toUpperCase() + text.substring(1)
+
 const breakCapitalWordsZeroWidth = (text) =>
   text.replace(/([A-Z])/g, "\u200B$1") // Adds a zero-width space before each capital letter. This way, the css word-break: break-word; rule can apply correctly
 
+const addFunctionTypeSymbol = (text, type) =>
```

Comment:
Can we have this function just return the function call operator instead of concatenating it?

`getFunctionCallOperator`, take function type, just return `:` or `.`